teammapper-frontend/mmp/src/map/handlers/nodes.ts

```diff
@@ -287,7 +287,9 @@ export default class Nodes {
                 Log.error('The property does not exist')
         }
         if (graphic === false && updated !== false) {
```

Comment:
please lets try to avoid so much if nesting. Example how it could be flatter:

```
  if (graphic === false && updated !== false && updateHistory) {
  this.map.history.save()
}

if (graphic === false && updated !== false && notifyWithEvent) {
  this.map.events.call(Event.nodeUpdate, node.dom, { nodeProperties: this.getNodeProperties(node), changedProperty: property, previousValue })
  }
```

Even better, create new methods that check for the conditions (remember to put these anonymous functions above the updateNode method or make real functions of of them, otherwise it won't work):

```
const shallUpdateWithEvent = () => graphic === false && updated !== false && notifyWithEvent;
const shallUpdateWithHistory= () => graphic === false && updated !== false && updateHistory
```